src/Microsoft.PowerShell.Commands.Management/commands/management/GetChildrenCommand.cs

```diff
@@ -172,6 +172,25 @@ public uint Depth
         }
 
         /// <summary>
+        /// Gets or sets whether symbolic links are to be followed when recursing.
+        /// </summary>
+        [Parameter]
+        public SwitchParameter FollowSymlink
```

Comment:
Event without `-FollowSymlink`, the cmdlet will follow symlinks specified in `-Path`